substratevm/mx.substratevm/testhello.py

```diff
@@ -453,6 +457,10 @@ def test():
     # print details of Hello type
     exec_string = execute("ptype 'hello.Hello'")
     rexp = [r"type = class hello\.Hello : public java\.lang\.Object {",
+            # r"%spublic:"%spaces_pattern,
+            # r"%sstatic %sjava\.util\.function\.Supplier \*lambda;"%(spaces_pattern, "_z_\." if isolates else ""),
```

Comment:
Why is this commented out?

Reply:
Accidentally, good catch! Do you want me to fix it here?

Reply:
Yes please. :)

Reply:
done